src/inverseRobot.js

```diff
@@ -24,7 +24,31 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const invRobot = {};
```

Comment:
Don't shorten variable names if it's unnecessary
```suggestion
  const inversedRobot = {};
```